docs/user/ppl/admin/catalog.rst

```diff
@@ -0,0 +1,86 @@
+.. highlight:: sh
+
+=================
+Catalog Settings
+=================
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 1
+
+Introduction
+============
+
+The concept of ``catalog`` is introduced to support the federation of SQL/PPL query engine to multiple data sources.
+This helps PPL users to leverage data from multiple data sources and derive correlation and insights.
+Catalog definition provides the information to connect to a datasource and also gives a name to them to refer in PPL commands.
+
+
+Definitions of catalog and connector
+====================================
+* Connector is a component that adapts the query engine to a datasource. For example, Prometheus connector would adapt and help execute the queries to run on Prometheus data source. connector name is enough in the catalog definition json.
+* Catalog is a construct to define how to connect to a datasource and which connector to adapt by query engine.
+
+Example Prometheus Catalog Definition ::
+
+    [{
+        "name" : "prometheus",
+        "connector": "prometheus",
+        "uri" : "http://localhost:9090",
+        "authentication" : {
+            "type" : "basicauth",
+            "username" : "admin",
+            "password" : "admin"
+        }
+    }]
+Catalog configuration Restrictions.
+
+* ``name``, ``uri``, ``connector`` are required fields in the catalog configuration.
+* All the catalog names should be unique.
+* Catalog names should match with the regex of an identifier[``[@*A-Za-z]+?[*a-zA-Z_\-0-9]*``].
+* ``prometheus`` is the only connector allowed.
+
+Configuring catalog in OpenSearch.
+====================================
+
+* Catalogs are configured in opensearch keystore as secure settings under ``plugins.query.federation.catalog.config`` key as they contain credential info.
+* A json file containing array of catalog configurations should be injected into keystore with the above mentioned key. sample json file can be seen in the above section.
+
+
+[``To be run on all the nodes in the cluster``] Command to add catalog.json file to OpenSearch Keystore ::
```

Comment:
why is "To be run on all the nodes in the cluster" code in brackets?

Reply:
can be a common mistake while updating settings. Running on one node and expecting to be reflected when we hit reload settings api. Any other idea to emphasize it?

Reply:
maybe use normal or bold text? i think formatting it as inline code is a bit confusing